src/ngraph/builder/quantization.cpp

```diff
@@ -121,13 +121,20 @@ namespace ngraph
                                            const std::shared_ptr<Node> max_freezed_output,
                                            const bool with_relu)
         {
+            bool is_output_signed = !with_relu;
             float scale = builder::quantization_util::get_scale(min_input,
                                                                 max_input,
                                                                 min_filter,
                                                                 max_filter,
                                                                 min_freezed_output,
-                                                                max_freezed_output);
+                                                                max_freezed_output,
+                                                                is_output_signed);
             auto requantization_scale = op::Constant::create(element::f32, Shape{1}, {scale});
+            float scale_bias = (bias->get_element_type() == element::i32)
+                                   ? 1.0
+                                   : builder::quantization_util::get_bias_scale(
+                                         min_input, max_input, min_filter, max_filter);
+            auto bias_scale = op::Constant::create(element::f32, Shape{1}, {scale_bias});
```

Comment:
Can we use a quantize op here to convert bias to int32?

Reply:
@jbobba are you talking about an nGraph::op::Quantize?